recipes/xpdtools/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "xpdtools" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "d730756e96368bd050a16d07a4e1f0563789aabbb753dec2dba7be29728ae3b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/xpdAcq/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - scikit-beam
+    - pyyaml
+    - pyFAI
+    - fire
+    - pims
+    - fabio
```

Comment:
Not sure if one of the other dependencies grabs it, but it looks like you also need `matplotlib` and `streamz`:

https://github.com/xpdAcq/xpdtools/blob/master/xpdtools/cli/process_tiff.py#L11

I saw the comment about waiting on streamz, but it looks like there is already a package for it:

https://anaconda.org/conda-forge/streamz/files

Reply:
I'm waiting on the next release (`starmap` only got in recently) https://github.com/mrocklin/streamz/issues/136